crates/test-utils/src/deploy_helpers.rs

```diff
@@ -831,6 +746,71 @@ impl KakarotTestEnvironmentContext {
         Self { sequencer, kakarot_client, kakarot, kakarot_contract, evm_contracts }
     }
 
+    /// Initializes the Kakarot test environment and deploys the Kakarot system and EVM contracts.
+    async fn initialize_without_dumped_state(
+        sequencer: TestSequencer,
+        starknet_provider: Arc<JsonRpcClient<HttpTransport>>,
+    ) -> Self {
+        let starknet_account = sequencer.account();
+
+        // Define the expected funded amount for the Kakarot system
+        let expected_funded_amount = FieldElement::from_dec_str("1000000000000000000").unwrap();
+
+        // Deploy the Kakarot system
+        let kakarot = deploy_kakarot_system(&sequencer, EOA_WALLET.clone(), expected_funded_amount).await;
+
+        let starknet_deployer_account = deploy_deployer_account(starknet_provider.clone(), &starknet_account).await;
```

Comment:
does this `deploy_deployer_account` work only for such a "deployer" account? and not "any" account?

Reply:
not sure to understand, I think we create the deployer account from any starknet account and use it to instantiate `KakarotClient`

Reply:
the name of the function `deploy_deployer_account` means a priori that the returned account is not a standard starknet account. What does this function make that is special to this deployer role that we have in the RPC? and if this function just creates a regular starknet account, then it's not a "deployer" account but rather

```
let deployer_account = deploy_account(starknet_provider.clone(), ...
```

Reply:
So this deployer account is a starknet account that the client uses to deploy EOAs. Looking into the function, what makes it different is that this gets funded so it's able to create EOAs so I think it makes sense to keep it

Reply:
i think in that sense it makes sense to create an issue and separate it into two functions: `deploy_account` and `fund_account`

Reply:
I would be fine with this but maybe I'm missing some context on this function and @bajpai244 could share more

Reply:
sure! @bajpai244 could you create an issue to perform the above changes?